### DISCUSSION TITLE
Search doesn't seem to work on my install

### DISCUSSION THREAD

**Discussion by tacticurn** [Alexco500/sidplay5#21] [Q&A]
When I type anything in the Search box, nothing shows up at all. I have downloaded HVSC '78 and can find plenty of sid tunes on my disk, and they all play back just fine. It's just not possible to use the search feature.

Any tips to what might be the problem?

I am running on 5.0.41 beta, but I also tried the stable 5.0 and even the old 4.2. MacOS Ventura 13.3.1

**Comment by Alexco500** [comment]
It seems that something is not working with your Spotlight setup. Does Spotlight in general work on your system? If not, can you try to rebuild the indexes (via macOS, see https://support.apple.com/en-us/HT201716)?
If spotlight is working correctly, can you please try to reindex the SIDs with this terminal/shell command:
`mdimport -r /Applications/SIDPLAY.app/Contents/Library/Spotlight/SIDMusic.mdimporter`
(if Sidplay is installed in /Applications, otherwise you need to modify the path).

**Comment by tacticurn** [comment]
Hi, thanks for the advice.

I tried to rebuild the Spotlight index by adding the C64Music folder with the HVSC content, then removing it again as described by Apple.

Then I ran the command you typed by pasting it into Terminal, and it seemed to execute (no response, but no error at least.)

This hasn't so far changed anything. But I notice that Spotlight doesn't want to show any sid files even when I search, until I have opened them once and accepted the warning that MacOS prompts, because the developer of file is "unknown". It apparently has something to do with MacOS Ventura's Gatekeeper.

**Comment by Alexco500** [comment]
Hm. Do you still use the Beta version? Because the Beta version is not notarized by Apple, but normal releases are. So you shouldn't see such kind of behaviour if you try that with a release version.

**Comment by tacticurn** [comment]
About says Version 5.0.1 (44), its from april 30th I think.

**Reply by Alexco500** [reply]
Does it show Beta in the window title?

**Comment by tacticurn** [comment]
Not that I can see. I have added a screensnip of the app open with the About window.
<img width="822" alt="Skærmbillede 2023-07-12 kl  19 51 38" src="https://github.com/Alexco500/sidplay5/assets/97593630/b22bf5f5-6c73-4a9a-a470-800f65ac8d70">

**Reply by Alexco500** [reply]
Ok, that looks like a normal release. Strange. How does it look in the Finder window? What happens if you double-click on a sid file?

**Comment by tacticurn** [comment]
Warns me that it is from an unknown developer:

<img width="261" alt="Screenshot 2023-07-12 at 21 50 15" src="https://github.com/Alexco500/sidplay5/assets/97593630/8903a208-f2bc-468d-83c8-6a07976f5885">

**Reply by Alexco500** [reply]
I am out of ideas at the moment. Did you try to delete the app (empty trashcan as well) and reinstall it?
